test/integration/DataUnionEndpoints/DataUnionEndpoints.test.js

```diff
@@ -83,7 +84,8 @@ describe('DataUnionEndPoints', () => {
                 await adminClient.kick(memberAddressList.slice(1), { dataUnion })
             })
             const res = await adminClient.getDataUnionStats({ dataUnion })
-            expect(+res.memberCount).toEqual(1)
+            expect(+res.activeMemberCount).toEqual(1)
```

Comment:
@jtakalai What's this `+variable` thing?

Reply:
@kare Coercing to variable number